modules/tinymce/CHANGELOG.md

```diff
@@ -8,152 +8,153 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Moved the `hr` plugin's functionality to TinyMCE core #TINY-8313
 - Moved the `print` plugin's functionality to TinyMCE core #TINY-8314
+- Moved non-UI table functionality to core #TINY-8273
+- The `DomParser` API no longer uses a custom parser internally and instead uses the native `DOMParser` API #TINY-4627
+- The `editor.getContent()` API can provide custom content by preventing and overriding `content` in the `BeforeGetContent` event. This makes it consistent with the `editor.selection.getContent()` API #TINY-8018
+- The `editor.setContent()` API can now be prevented using the `BeforeSetContent` event. This makes it consistent with the `editor.selection.setContent()` API #TINY-8018
+- Add-ons such as plugins and themes are no longer constructed using the `new` operator #TINY-8256
+- A number of APIs that were not proper classes, are no longer constructed using the `new` operator #TINY-8322
+- The Editor commands APIs will no longer fallback to executing the browsers native command functionality #TINY-7829
+- The Editor query command APIs will now return `false` or an empty string on removed editors #TINY-7829
+- The `mceAddEditor` and `mceToggleEditor` commands now take an object as their value to specify the id and editor options #TINY-8138
+- The `mceInsertTable` command can no longer open the insert table dialog. Use the `mceInsertTableDialog` command instead #TINY-8273
+- The `plugins` option now returns a `string` array instead of a space separated string #TINY-8455
 - The `media` plugin no longer treats `iframe`, `video`, `audio` or `object` elements as "special" and will validate the contents against the schema #TINY-8382
+- The `images_upload_handler` option is no longer passed a `success` or `failure` callback and instead requires a `Promise` to be returned with the upload result #TINY-8325
+- The `tinymce.settings` global property is no longer set upon initialization #TINY-7359
 - The `change` event is no longer fired on first modification #TINY-6920
-- Renamed the `getShortEndedElements` Schema API to `getVoidElements` #TINY-8344
-- Changed the default statusbar element path delimiter from `»` to `›` #TINY-8372
+- The `GetContent` event will now always pass a `string` for the `content` property #TINY-7996
 - Changed the default tag for the strikethrough format to the `s` tag when using a html 5 schema #TINY-8262
 - The `strike` tag is automatically converted to the `s` tag when using a html 5 schema #TINY-8262
+- Aligning a table to the left or right will now use margin styling instead of float styling #TINY-6558
+- The `:` control character has been changed to `~` for the schema `valid_elements` and `extended_valid_elements` options #TINY-6726
+- The `primary` property on dialog buttons has been deprecated. Use the new `buttonType` property instead #TINY-8304
+- Changed the default statusbar element path delimiter from `»` to `›` #TINY-8372
+- Replaced 'Powered by Tiny' link text with logo #TINY-8371
```

Comment:
```suggestion
- Replaced the `Powered by Tiny` link text with the Tiny logo #TINY-8371
```
Backticks instead of quotes for this maybe? Also I don't think this fully made sense, so made a minor change there.